js/sound.js

```diff
@@ -55,7 +55,14 @@ const backgroundAudio = [
         legalNotice: "Music by Saarthak Chaturvedi",
         link: "http://github.com/saarthakchats",
     },
-        {
+    {
+        filename: "audio/art/musicofart-backgvideo.mp3",
```

Comment:
Now it's good but audio.mp3 is of no use, please delete it. :

Reply:
No audio.mp3 file

Reply:
I removed it already

Reply:
Its still there

Reply:
Yep

Reply:
@arjunpandey2003 no there are two sound files beacuse I added background music and tile sound both please see the caption.